contrib/ci/fwupd_setup_helpers.py

```
@@ -293,17 +308,19 @@ def install_packages(
         packages = get_build_dependencies(profile, variant, cross)
     installer = _get_installer_cmd(profile, yes)
     installer += packages
-    if debugging:
-        print(installer)
+    logger.debug(installer)
     try:
         subprocess.check_output(installer, stderr=subprocess.STDOUT)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)